### PR TITLE
Profile Addition

### PR DESCRIPTION
**Pull request by mohit1607**
Addition of new profile mohit1607.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2532"></a>

### CODE REVIEW

**Review by senali-d — COMMENTED** [review]
Looks great.

File: data/mohit1607.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "MOHIT ASHLIYA",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Open Source Enthusiast with taste in React!",
+  "avatar": "https://github.com/mohit1607.png",
+  "tags": ["javascript", "typescript", "react.js", "node.js", "mongodb", "fullstack"],
+  "socials": [
+    { "icon": "twitter", "url": "https://twitter.com/ashliya_mohit" },
+    { "icon": "github", "url": "https://github.com/mohit1607" }
```
Comment (eddiejaoude) [inline data/mohit1607.json:10]: These are not icons, please use icons available https://linkfree.eddiehub.io/icons

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Icons need updating, inline comment added

**Comment by mohit1607** [comment]
Hi Eddie thanks for reviewing my profile and Now I look forward to solve more issues.

**Review by Pradumnasaraf — APPROVED** [review]

**Review by Pradumnasaraf — COMMENTED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Pull request merged**